go.mod

```diff
@@ -51,6 +50,7 @@ require (
 	github.com/sagikazarmark/locafero v0.4.0 // indirect
 	github.com/sagikazarmark/slog-shim v0.1.0 // indirect
 	github.com/sahilm/fuzzy v0.1.1 // indirect
+	github.com/sirupsen/logrus v1.9.3 // indirect
```

Comment:
Weird that `logrus` is still here as an indirect dependency. Is it a dependency of `slog`?

Reply:
`go mod graph | grep logrus` tells me that it's `github.com/ubuntu/decorate` which pulls in this indirect dependency

Reply:
We only have a single usage of any of the functions in `github.com/ubuntu/decorate` which depend on logrus (that's https://github.com/ubuntu/authd/blob/main/cmd/authd/daemon/daemon.go#L142-L142). To have consistent logs, I propose we:
* Replace that usage of `LogOnError` with a simple `if err != nil { log.Warning(context.TODO(), err.Error()) }`
* Copy the implementation of `decorate.OnError` (which has many usages in the authd repo) into a package in our own repo
* Remove the dependency on `github.com/ubuntu/decorate`, so that we're not tempted to use its functions which use `logrus` for logging

Reply:
Since we own that repo, I'd go with removal of the dependency from there instead of reimplementing it here, no?

Reply:
I didn't check which other repos are using it, and how they handle logging. I don't think it's a good idea to use any non-standard logging in an external library, because you don't know which logging module the caller uses. Since `LogOnError` is not doing anything else, I don't think it's useful to define this function inside an external library.